vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -19,7 +19,7 @@
 else:
     fused_experts = None  # type: ignore
 if current_platform.is_tpu():
-    from .moe_pallas import fused_moe as fused_moe_pallas
+    from .moe_torch_iterative import fused_moe as fused_moe_pallas
```

Comment:
Could you leave a comment that moe_pallas should be used once issue is resolved?